packages/core/src/App/Components/Layout/Header/platform-switcher.jsx

```diff
@@ -47,10 +47,12 @@ class PlatformSwitcher extends React.PureComponent {
                 >
                     <Icon
                         className='platform-switcher__icon'
-                        icon={getPlatformIcon(this.props.app_routing_history)}
+                        icon={getPlatformInformation(this.props.app_routing_history).icon}
```

Comment:
Is `icon` always available? Don't we need to check it?

Reply:
yes, that's a fixed property which is always available.